test/e2e/konnect_test.go

```diff
@@ -39,7 +39,7 @@ import (
 
 const (
 	konnectRuntimeGroupsBaseURL          = "https://us.kic.api.konghq.tech/v2"
-	konnectRuntimeGroupsConfigBaseURLFmt = "https://us.api.konghq.tech/konnect-api/api/runtime_groups/%s/v1"
+	konnectRuntimeGroupsConfigBaseURLFmt = "https://us.api.konghq.tech/konnect-api/v2/control_planes/%s/"
```

Comment:
According to Dave's suggestion, I think it should go with a dash instead of an underscore.
```suggestion
	konnectRuntimeGroupsConfigBaseURLFmt = "https://us.api.konghq.tech/konnect-api/v2/control-planes/%s/"
```